05 SimpleApp_Navigation/src/pages/login/actions/loginRequestStarted.ts

```diff
@@ -0,0 +1,25 @@
+import { actionsEnums } from '../../../common/actionsEnums';
+import { LoginEntity } from '../../../model/login';
+import { loginApi } from '../../../rest-api/loginApi';
+import { loginRequestCompletedAction } from './loginRequestCompleted';
+import { hashHistory } from 'react-router';
+
+export const loginRequestStartedAction = (login: LoginEntity) => {
+    return function (dispatcher) {
+        const promise = loginApi.login(login);
+
+        promise.then(
+            data => {
+                dispatcher(loginRequestCompletedAction(data));
```

Comment:
Refactor this move to the same file the started and completed requests.

actions/loginRequest.ts